Godeps/_workspace/src/github.com/openshift/openshift-sdn/pkg/ovssubnet/controller/multitenant/bin/openshift-sdn-multitenant-setup.sh

```diff
@@ -114,6 +120,7 @@ function setup() {
     ip link set vovsbr up
     ip link set vlinuxbr txqueuelen 0
     ip link set vovsbr txqueuelen 0
+    brctl addif lbr0 vlinuxbr
```

Comment:
> What if lbr0 gets deleted by chance

What if eth0 loses its IP address by chance. There are always going to be ways the system could get messed up by admins running random commands... The script is mostly only trying to deal with the case of "openshift is restarted and the config is different than it was before".